# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../silo/ls-c-include-order.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./second/ls.c.orig	2004-06-28 21:28:22.725307784 +0200
+++ ./second/ls.c	2004-06-28 21:28:29.333303216 +0200
@@ -19,10 +19,10 @@
    USA.  */
 
 #include <silo.h>
+#include <linux/types.h>
 #include <stringops.h>
 
 typedef int FILE;
-#include <linux/types.h>
 #include <ext2fs/ext2_fs.h>
 #include <ext2fs/ext2fs.h>
 
